ec2/tags.go

```diff
@@ -0,0 +1,44 @@
+package ec2
+
+import (
+	"context"
+
+	"github.com/YaleSpinup/apierror"
+	"github.com/YaleSpinup/ec2-api/common"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	log "github.com/sirupsen/logrus"
+)
+
+func (e *Ec2) UpdateTags(ctx context.Context, id string, rawTags map[string]string) error {
+	if id == "" || len(rawTags) == 0 {
+		return apierror.New(apierror.ErrBadRequest, "invalid input", nil)
+	}
+	resources := e.getResourcesById(id)
+	tags := toEc2Tags(rawTags)
+
+	log.Infof("updating resources: %v with tags %+v", resources, tags)
+
+	input := ec2.CreateTagsInput{
+		Resources: resources,
```

Comment:
You don't need this variable, can just put `[]*string{aws.String(id)}` here

Reply:
To account for the case for there are more than one resource.

Reply:
If you make ids a slice, then you can just do `aws.StringSlice(ids)` here, and get rid of the `getResourcesById` function

Reply:
Done.